src/index.html

```diff
@@ -28,6 +28,12 @@
       </ul>
     </article>
   </div>
+  <div class="buttons-container">
+    <a href="#" class="btn">get started</a>
+
+    <button class="form-btn">join the ranks</button>
```

Comment:
Okay so before you make a form button just make a primary button. So these styles you've applied at the moment to `.form-btn` should be generic for say a `primary-btn`.
A `button` element should inherit the styles for the primary btn by default without having to add a class. So do something like this:

```css
button, .primary-btn {
  // Styles for a default button here
}
```

```html
<button> Register </button>
<a class="primary-btn" href="#"> Enter </a>
```

This is the final effect we are looking for ^